portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.path

```diff
@@ -0,0 +1,31 @@
+<html>
+<head>
+<title>MyTestFile</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">MyTestFile</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>APPLICATIONS_MENU</td>
+	<td>//button[@data-qa-id='applicationsMenu']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CONTROL_PANEL</td>
+	<td>//li[contains(@id,'ApplicationsMenuPortlet_tab_2')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CONFIGURATION</td>
```

Comment:
Hey @crislucas1, I think you could put another name here

Remember that the name of the tests, macros and xpaths must be clear so that when a person reads the name, he understands what that test, macro or xpath does. In your case you used `CONFIGURATION`, I think that on the portal or on any platform this term can be present in several places. So it's important to specify a little more, like: `CONFIGURATION_STRING`, `CONFIGURATION_LABEL`, `CONFIGURATION_TERM` and etc.

That way the person reading it will understand that this xpath finds the word configuration that is present on the screen.

You specified well in the name of the other xpaths, as you named with the buttons name for them and this will be clear to a person who will read the name and have knowledge about the DXP Portal

Reply:
Ok Vinicius, yes i can, i will rename them. Thanks for the feedback